go.mod

```diff
@@ -3,6 +3,12 @@ module github.com/gravitational/ttlmap
 go 1.13
```

Comment:
I thought the proper way of introducing a v2 for a module was changing the module path:
```
module github.com/gravitational/ttlmap/v2

go 1.13
...
```
I would have a `v1` for the older version to import like this:
```go
import "github.com/gravitational/ttlmap" // legacy version
```
and a `v2` branch that would be a proper go module v2:
```go
import "github.com/gravitational/ttlmap/v2"
```